frontend/src/pages/ActivityReport/Pages/activitySummary.js

```diff
@@ -486,9 +525,33 @@ const sections = [
   },
 ];
 
-const ReviewSection = () => (
-  <ReviewPage sections={sections} path="activity-summary" />
-);
+const ReviewSection = () => {
+  const { watch } = useFormContext();
+  const {
+    context,
+    calculatedStatus,
+  } = watch();
+
+  const canEdit = reportIsEditable(calculatedStatus);
+  return (
+    <>
+      <ReviewPage sections={sections} path="activity-summary" />
+      <Section
+        hidePrint={isUndefined(context)}
+        key="context"
+        basePath="goals-objectives"
```

Comment:
This needs to be updated to `activity-summary`

Reply:
Nice catch, will update.